tools/tgs4_scripts/PreCompile.bat

```diff
@@ -0,0 +1,14 @@
+@echo off
+set TG_BOOTSTRAP_CACHE=%cd%
+IF NOT "%1" == "" (
+	rem TGS4: we are passed the game directory on the command line
+	cd %1
+) ELSE IF EXIST "..\Game\A\tgstation.dmb" (
+	rem TGS3: Game/A/tgstation.dmb exists, so build in Game/B
+	cd ..\Game\B
+) ELSE (
+	rem TGS3: Otherwise build in Game/A
+	cd ..\Game\A
```

Comment:
I got the priority order backwards here for when the .dmb doesn't exist at all
```suggestion
) ELSE IF EXIST "..\Game\B\tgstation.dmb" (
	rem TGS3: Game/B/tgstation.dmb exists, so build in Game/A
	cd ..\Game\A
) ELSE (
	rem TGS3: Otherwise build in Game/B
	cd ..\Game\B
```